app/src/main/java/com/teamagam/gimelgimel/app/map/viewModel/IMapView.java

```diff
@@ -0,0 +1,16 @@
+package com.teamagam.gimelgimel.app.map.viewModel;
+
+import com.teamagam.gimelgimel.app.map.model.VectorLayer;
+import com.teamagam.gimelgimel.app.map.model.geometries.PointGeometry;
+
+/**
+ * Created on 8/18/2016.
+ * TODO: complete text
+ */
+public interface IMapView {
+    void takePicture();
```

Comment:
I totally agree with you. this is just to keep the flow for the takeImage User story.